datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/dataset/mappers/EditableSchemaFieldInfoMapper.java

```diff
@@ -0,0 +1,32 @@
+package com.linkedin.datahub.graphql.types.dataset.mappers;
+
+import com.linkedin.datahub.graphql.types.mappers.ModelMapper;
+import com.linkedin.datahub.graphql.types.tag.mappers.GlobalTagsMapper;
+import com.linkedin.schema.EditableSchemaFieldInfo;
+
+import javax.annotation.Nonnull;
+
+
+public class EditableSchemaFieldInfoMapper implements ModelMapper<EditableSchemaFieldInfo, com.linkedin.datahub.graphql.generated.EditableSchemaFieldInfo> {
+
+    public static final EditableSchemaFieldInfoMapper INSTANCE = new EditableSchemaFieldInfoMapper();
+
+    public static com.linkedin.datahub.graphql.generated.EditableSchemaFieldInfo map(@Nonnull final EditableSchemaFieldInfo fieldInfo) {
+        return INSTANCE.apply(fieldInfo);
+    }
+
+    @Override
+    public com.linkedin.datahub.graphql.generated.EditableSchemaFieldInfo apply(@Nonnull final EditableSchemaFieldInfo input) {
+        final com.linkedin.datahub.graphql.generated.EditableSchemaFieldInfo result = new com.linkedin.datahub.graphql.generated.EditableSchemaFieldInfo();
+        if (input.hasDescription()) {
```

Comment:
Are these null checks required? If input.description() is null, won't it just set the output description to null.. anything wrong with this?

Reply:
repeating above, I didn't dig into this too much but yes, I did get null pointer exceptions without it